src/logging/LogManager.java

```diff
@@ -0,0 +1,14 @@
+package logging;
+
+public class LogManager {
+
+    public static boolean isLoggingEnabled;
+
+    public static void enableLogging(boolean enable) {
+        isLoggingEnabled = enable;
+    }
+
+    public static boolean isLoggingEnabled() {
+        return isLoggingEnabled;
+    }
```

Comment:
Don't use `Static` ! 

Best practice is to use Singleton.

Reply:
done